Docs/Configuration.tex

```diff
@@ -3108,7 +3098,7 @@ \subsection{Boot Properties}\label{miscbootprops}
     \emph{Note 1}: Apple's boot picker partially supports placing a volume icon file
     at the operating system's \texttt{Data} volume root (\texttt{/System/Volumes/Data/} when
     mounted at default location within macOS). This approach is broken: that file is not
-    accessible either by OpenCanopy or by Apple's own boot picker when FileVault 2 is
+    accessible either by OpenCanopy or by Apple's own boot picker when FaultVault 2 is
```

Comment:
It is FileVault, not FaultVault.

Reply:
Actually thought I saw a commit fixing this earlier and rebased on current status.
Anyway, fixed now